RocksDBStore/Plugins/Storage/Settings.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.Extensions.Configuration;
+
+namespace Neo.Plugins.Storage
+{
+    internal class Settings
```

Comment:
Good @shargon, great job!
This file will be moved replicated on LevelDbStore as well?

Reply:
Yes, in order to use the same structure